src/Renci.SshNet/Security/Cryptography/RsaDigitalSignature.cs

```diff
@@ -41,13 +41,20 @@ public RsaDigitalSignature(RsaKey rsaKey, HashAlgorithmName hashAlgorithmName)
         /// </returns>
         protected override byte[] Hash(byte[] input)
         {
-            return _hash.ComputeHash(input);
+#if !NET462
+            using var hash = IncrementalHash.CreateHash(_hashAlgorithmName);
+            hash.AppendData(input);
+            return hash.GetHashAndReset();
```

Comment:
Overall this PR looks good, but I need to understand this change before I could be able to merge this.

Reply:
Sure. `IncrementalHash` is a more modern and slightly lighter-weight method of hashing than `HashAlgorithm`. And it doesn't have trimming problems like `CryptoConfig.CreateFromName` does.